alert-policies/amazon-workmail/IncomingEmailBounced.yml

```diff
@@ -0,0 +1,27 @@
+name: High IncomingEmailBounced
```

Comment:
```suggestion
name: High Incoming Email Bounced
```

Reply:
@josephgregoryii , Addressed the changes